native/src/modules/common/components/ProgressBar.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import * as React from 'react'
+import Svg, { Circle, Image } from 'react-native-svg'
+import logo from './assets/app-logo-g.svg'
+import { Dimensions } from 'react-native'
+import styled from 'styled-components/native'
+import { type StyledComponent } from 'styled-components'
+
+const Container: StyledComponent<{||}, {||}, *> = styled.View`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+const Text = styled.Text`
+  padding-top: 25px;
+  font-size: 20px;
+  font-weight: 700;
+`
+
+const { width } = Dimensions.get('window')
+const size = width / 3 // eslint-disable-line no-magic-numbers
+
+const logoSize = size * 0.6 // eslint-disable-line no-magic-numbers
+const logoXY = (size - logoSize) / 2
+const logoToCenter = `translate(${logoXY}, ${logoXY})`
+
+// progress
+const strokeWidth = size * 0.09 // eslint-disable-line no-magic-numbers
```

Comment:
Would be much nicer to use constants instead of disabling eslint.

Reply:
Yes, I kind of like it to explicitly name all values. What is 0.09 and why was it choosen?

That can be done by doing:

```
const STROKE_WIDTH_FRACTION = 0.09
```

Or maybe it also makes sense to just use a value like 10 for the strokeWidth.

Reply:
I wanted to keep it proportional so it automatically scales with the width of the screen

I used constants now